src/scss/main.scss

```diff
@@ -88,7 +88,7 @@ h1 {
 }
 
 h2 {
-    @apply text-3xl;
+    font-size: 32px;
```

Comment:
I switched from tailwind to coding in 32px since that is what we want to use for H2s now and tailwind doesn't have a style that fits that font-size.